_posts/areiter18/2021-12-08-Final-Project-areiter18.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+author: areiter18
+title: Andrew R's Final Project Code + Reflection
+
+---
```

Comment:
Andrew, could you enclose the title in `"`s to avoid that apostrophe causing grief? I also recommend removing the extra whitespace between the title and the close of the header (`---`).